### ISSUE TITLE
node_mapping / phase_mapping could be removed to simplify code

### ISSUE THREAD

**Issue by EveCharbie** [pyomeca/bioptim#717]

**Comment by Ipuch** [comment]
Can you give a little overview of the code area expected to be touched ?

**Comment by Ipuch** [comment]
DONE !

**Comment by Ipuch** [comment]
I still have a bunch of questions about these functions @EveCharbie  : 

https://github.com/pyomeca/bioptim/blob/39ed19797e796eedc2745f10db64795b0e3117a4/bioptim/dynamics/configure_problem.py#L121-L138

https://github.com/pyomeca/bioptim/blob/39ed19797e796eedc2745f10db64795b0e3117a4/bioptim/dynamics/configure_problem.py#L1883-L1919

That's only two right but there might have been some stuff hidden in the plot section right ?

**Comment by EveCharbie** [comment]
Yes, I will fix this when I do the refactor on ode_solver since lots of thing declared in ocp can be moved to nlp :)
In addition, I think it would be possible to decouple the ode_solver from the StochasticBiorbdModel definition and uniformize this particular case with the normal dynamics declaration.
